front/src/hooks/useQrResolver.ts

```diff
@@ -79,22 +84,44 @@ export const useQrResolver = () => {
               qrHash: hash,
             } as IQrResolvedValue;
           }
+          // @ts-ignore
           if (!data?.qrCode?.box) {
```

Comment:
missing cases are:

- NO_BOXTRIBUTE_QR: `data.qrCode.__typename === "ResourceDoesNotExistError" --> "No box found for this QR code!" toast (Currently line 65)
- NOT_AUTHORIZED_FOR_QR: `data.qrCode.__typename === "InsufficientPermissionError" --> "You don't have permission to access this box!" toast (Currently line 56)